src/server/index.spec.tsx

```diff
@@ -376,29 +394,38 @@ describe(__filename, () => {
           );
 
           // This Express app is used to simulate the API service.
-          app = express();
+          apiApp = express();
           // We create a handler to configure a response without cookies.
-          app.get('/api/no-cookie', (req, res) => {
+          apiApp.get('/api/no-cookie', (req, res) => {
             res.send(apiResponseBody);
           });
           // This is a catch-all handler for all other requests.
-          app.get('*', (req, res) => {
+          apiApp.get('*', (req, res) => {
             res.cookie('auth_token', 'secret', {
               // Cookies returned by the API are usually secure.
               secure: true,
             });
             res.send(apiResponseBody);
           });
 
-          fakeApiServer = app.listen(apiPort);
+          fakeApiServer = apiApp.listen(apiPort);
           fakeApiServer.on('error', (error) => {
             // eslint-disable-next-line no-console
             console.error('proxy error', error);
           });
+
+          // This Express app is used to simulate the reviewers service.
+          reviewersApp = express();
+          reviewersApp.get('*', (req, res) => {
+            res.send(reviewersResponseBody);
+          });
+
+          fakeReviewersServer = reviewersApp.listen(reviewersPort);
```

Comment:
We should probably add a listener on `error` like it's done for the `fakeApiServer`.

Reply:
Why do we need one? "Just in case"?

Reply:
yep, in case the server fails to start for example.